OpenRVS/classes/OpenServerList.uc

```diff
@@ -64,14 +65,21 @@ function ParseServers(OpenHTTPClient.HttpResponse resp)
 		return;
 	}
 
-	// Determine response type.
+	class'OpenLogger'.static.Info("loading servers from Internet", self);
 	class'OpenString'.static.Split(resp.Body, Chr(10), lines);
+	class'OpenLogger'.static.Info("parsing" @ lines.Length @ "server lines", self);
+
+	// Determine response type.
+	if (Left(lines[0], Len(CSV_HEADER_LINE)) == CSV_HEADER_LINE)

```

Comment:
Can we either call Caps() to compare all-capital letter versions of these strings, or use the boolean operator `~=` so that case doesn't matter?  I just want to make sure that we don't mess up our parsing if the server host decides to have "Name" instead of "name".